code/basic-transactions/calc-tx-cost/calc-tx-est-fees.preview.en.ts

```diff
@@ -0,0 +1,14 @@
+const recentBlockhash = await connection.getLatestBlockhash();
+
+const transaction = new Transaction({
+  recentBlockhash: recentBlockhash.blockhash
+})
+  .add(SystemProgram.transfer({
+    fromPubkey: payer.publicKey,
+    toPubkey: payee.publicKey,
+    lamports: 10
+  }));
+transaction.sign(payer);
```

Comment:
Do you have to sign first?

Reply:
I don't think so. Since that was in the original example, I figured it might be worth keeping it there since it's under the "Sending Transactions" section. I can go ahead and remove it.

Reply:
Let's just update this and we'll be good!

Reply:
Whoops, I took it out of the code but not the preview. Fixing that now.